src/common/components/nav/nav.component.js

```diff
@@ -174,20 +174,11 @@ class NavController{
     }))
       .map((response) => {
         let jsonStructure = response.data;
-        let menuStructure = {
-          main: jsonStructure['/content/cru/us/en'],
+        return {
+          main: jsonStructure['/content/cru/us/en'] || jsonStructure['main'],
           global: jsonStructure['/content/cru/us/en/global'],
           give: jsonStructure['/content/give/us/en']
         };
-
-        //add give to main nav
-        menuStructure.main.push({
-          title: 'Give',
-          path: '/give',
-          children: jsonStructure['/content/give/us/en']
-        });
```

Comment:
It's already in the json, "Hide in Nav" just needs to be unchecked.  It won't require any code changes.  I'm not sure why that was hard coded.